apps/catalogue/components/table/Content.vue

```diff
@@ -54,6 +56,20 @@ watch(orderByColumn, () => {
   fetchRows();
 });
 
+watch(
+  () => props.filter,
```

Comment:
is this really the appropriate syntax? I had hope vue would allow us something nicer like watch(props.filter,fetchRows())

Reply:
simple value wouldn't work? not very import but this makes my eyes bleed.